test/Faker/Provider/it_IT/CompanyTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Faker\Test\Provider\it_IT;
+
+use Faker\Generator;
+use Faker\Provider\it_IT\Company;
+
+class CompanyTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        $faker = new Generator();
+        $faker->addProvider(new Company($faker));
+        $this->faker = $faker;
+    }
+
+    public function testIfTaxIdCanReturnData()
+    {
+        $vatId = $this->faker->vatId();
+        $this->assertRegExp('/^IT{12}[0-9]$/', $vatId);
```

Comment:
I don't know exactly how the `regexify` method works, but this doesn't match an italian VAT :\